start-from-zero/index.css

```diff
@@ -44,27 +94,50 @@ body, html {
 form {
     display: flex;
     flex-direction: column;
-    
     background: white;
     padding-top: 30px;
     padding-bottom: 30px
 }
 
 .blue-line {
     text-align: center;
-    height: 50px;
-    line-height: 50px;
     background: #03A9F4;
+    height: 50px;
+    line-height: 48px;
     color: white;
+    font-size: 16px;
+    -webkit-box-shadow: 0px 3px 15px -2px rgba(0,0,0,0.79);
+    box-shadow: 0px 3px 15px -2px rgba(0,0,0,0.79);
+    position: relative;
+    z-index: 10000;
```

Comment:
The reason I'm asking is because it's much too easy to just come up with a random `z-index` value that can then clash with some other `z-index` in who knows what 3rd party styling assets or perhaps equally randomly chosen values in some other obscure area of the app that you might not even know about. I've actually seen this happen in (granted, gigantic) real-life applications, so it is something to be aware of and concerned about, especially when working on large scale projects.